src/index.js

```diff
@@ -144,7 +144,11 @@ export default class Video extends WdioReporter {
       return;
     }
 
-    this.addFrame();
+    // Skips screenshot if alert is displayed
+    browser.isAlertOpen().then((result) => {
+      if (!result) this.addFrame();
+      else helpers.debugLog('Skipped screenshot to avoid unexpected alert closing\n');
+    })
```

Comment:
```suggestion
    return browser.isAlertOpen().then((result) => {
      if (!result) {
          return this.addFrame();
      }
      
      return helpers.debugLog('Skipped screenshot to avoid unexpected alert closing\n');
    })
```